### ISSUE TITLE
[Azure Toolkit][A11yMAS][Accessibility][Keyboard][MAS2.1.1][IntelliJ][Wipro]: Unable to access "Context menu" with keyboard for Azure Explorer- Open ACR Explorer dialog.

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/azure-tools-for-java#1517]
**Repro Steps:**

1. In "Azure Explorer" , select the pushed image under container registries.
2. Try to access the "Context menu" using  "Shift + F10" with keyboard.

**Expected Result:**
User should be able to access the "Context menu" with keyboard.

**Actual Result:**
Unable to access "Context menu" with keyboard for Azure Explorer- Open ACR Explorer.

**Platform:**
IntelliJ IDEA 2017.3.4 (Community edition)
Build# IC-173.4127.27
JRE:1.8.0_152-release-1024-b8amd64
JVM: OpenJDK 64 Bit Server VM by Jetbrains.
**OSBuild Version:** Version 1709(Os Build 16299.192)

![open acr explorer context menu is not accessible](https://user-images.githubusercontent.com/22956130/36151012-7d3b1338-107a-11e8-8d3a-c7447213ad43.png)

**Comment by Eskibear** [comment]
duplicate with #1519

**Comment by WiproPSAccTest** [comment]
Keyboard user are not able to access the context menu .
Can you please provide the shortcut key to open the context menu

**Comment by Eskibear** [comment]
Will be fixed in PR #1638 , "CONTEXT_MENU" key is supposed to be pressed. (usually the one next to your right-win key)

**Comment by Eskibear** [comment]
Already in latest dev build.

**Comment by WiproPSAccTest** [comment]
Verified the issue in V3.7.0 .Issue got fixed .
Please add closed tag
![1517-fixed](https://user-images.githubusercontent.com/22956130/39859340-3125a348-5457-11e8-8ea8-54befccf4672.jpg)

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749978
